docs/md/content/usage/tool-authoring.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Authoring Tools"
+description: "Learn how to author a PRM tool."
+category: narrative
+tags:
+  - templates
+  - usage
+weight: 20
+---
+
+## Structure
+
+A PRM tool package can be packaged as a `.tar.gz` or hosted in a Github repository.
+It is comprised of a `prm-config.yml` and an optional `content` directory.
```

Comment:
```suggestion
It is comprised of a `prm-config.yml` and an `content` directory.
```
`content` dir is no longer optional

Reply:
This doesn't seem to be the case:

- I can still omit a `content` dir and it builds OK
- Including an empty `content` dir causes `docker build` to fail